packages/ui/src/components/form-builder/Forms/CreateBusiness/CreateBusinessForm.tsx

```
@@ -0,0 +1,118 @@
+		loading: "Carregando...",
+		nameRequired: "Nome é obrigatório e deve ter pelo menos 3 caracteres",
+		businessSlug: "Identificador Único",
+		businessNamelabel: "Nome do business",
+		businessNamePlaceholder: "Acme. S/A",
+		businessSluglabel: "Slug",
+		businessSlugPlaceholder: "Ex: acme-business-123",
+	},
+} as const;
+
+type LoginFormProps = {
+	mutate: (input: any) => void;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)